src/IO/ResampleLabelImage/resampleLabelImage.js

```diff
@@ -0,0 +1,58 @@
+import { runPipeline, InterfaceTypes, imageSharedBufferOrCopy } from 'itk-wasm'
+
+import itkConfig from '../itkConfig.js'
+
+async function runWasm(pipeline, args, image) {
+  const taskArgs = ['0', '0', ...args, '--memory-io']
+  const inputs = [
+    {
+      type: InterfaceTypes.Image,
+      data: imageSharedBufferOrCopy(image),
+    },
+  ]
+
+  const desiredOutputs = [{ type: InterfaceTypes.Image }]
+
+  const { pipelinesUrl, pipelineWorkerUrl } = itkConfig
+
+  // remove blob:http://... added to __webpack_public_path__ in webworker
+  const pipelinesUrlNoBlob = pipelinesUrl.startsWith('blob:')
+    ? pipelinesUrl.substring(5)
+    : pipelinesUrl
+
+  const pipelinesWorkerUrlNoBlob = pipelineWorkerUrl.startsWith('blob:')
+    ? pipelineWorkerUrl.substring(5)
+    : pipelineWorkerUrl
+
+  const worker = new Worker(
+    new URL(pipelinesWorkerUrlNoBlob, self.location.origin) // URL is invalid for tests unless base provided to URL
+  )
+
+  const { outputs } = await runPipeline(
+    worker,
+    pipeline,
+    taskArgs,
+    desiredOutputs,
+    inputs,
+    pipelinesUrlNoBlob
```

Comment:
We could not pass this argument -- since the wasm module will be available in the `itkConfig.pipelinesUrl` in this case. The `pipelinesUrl` argument is available as an alternative override.